akash/chain.json

```diff
@@ -79,6 +79,16 @@
       {
         "id": "47f7b7a021497ad7a338ea041f19a1a11ae06795",
         "address": "rpc.la.akash.farm:26656"
+      },
+      {
+        "id": "e1b058e5cfa2b836ddaa496b10911da62dcf182e",
+        "address": "akash-seed-1.allnodes.me:26656"
```

Comment:
Comma missing after 'address' in every file

Reply:
Thank you. Should be fixed now